src/app/log-tool/export-modal/export-modal.component.html

```diff
@@ -0,0 +1,40 @@
+<div bsModal #exportModal="bs-modal" class="modal fade" tabindex="-1" role="dialog" aria-labelledby="exportModalLabel"
+    aria-hidden="true" [config]="{backdrop: 'static'}">
+    <div class="modal-dialog modal-lg export-modal">
```

Comment:
We don't need such a big modal. the "modal-lg" class can probably be like "modal-md"